ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug, triage
+assignees: ''
+
+---
+
+<!-- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
```

Comment:
It's worth to move the `Current behavour` section upword and put it earlier than `Expected behaviour`. And probably rename it to the `Unexpected behaviour` or something like `Bug description`.

Reply:
`Current behaviour` is just fine, but moving it in front of expected one can be useful, I've reordered these in new bugs a number of times.